cmake/fake_tribits.cmake

```diff
@@ -94,6 +94,18 @@ FUNCTION(KOKKOS_ADD_TEST)
     # Prepend package name to the test name
     # These should be the full target name
     SET(TEST_NAME ${PACKAGE_NAME}_${TEST_NAME})
+
+    # For compatibility with Trilinos testing
+    IF(${TEST_NAME}_DISABLE)
+      RETURN()
+    ENDIF()
+    IF(${TEST_NAME}_SET_RUN_SERIAL)
+      SET_TESTS_PROPERTIES(${TEST_NAME} PROPERTIES RUN_SERIAL ON)
+    ENDIF()
+    IF(${TEST_NAME}_ENVIRONMENT)
```

Comment:
Maybe add a comment that TriBITS supports `_EXTRA_ARGS` instead of `_ENVIRONMENT`